src/cli/src/cluster/check.rs

```diff
@@ -477,7 +477,15 @@ fn check_helm_version() -> Result<(), IoError> {
 }
 
 fn check_sys_charts() -> Result<(), IoError> {
-    let sys_charts = install::installed_sys_charts(SYS_CHART_NAME);
+    let sys_charts = ClusterInstaller::sys_charts().map_err(|err| {
```

Comment:
this should return original error instead of mapping to IoError

Reply:
This is related to pre-flight command: `fluvio cluster check --pre-install` and hence using IoError.

Reply:
Same question as above

Reply:
ok. keep IoError for now